cabra/src/pages/components/TaskDetailsCard.tsx

```diff
@@ -65,6 +66,11 @@ export default function TaskDetails({ id }: Props) {
               <TrashIcon height={20} width={20} />
             </NavigationButton>
           </Ctas>
+          {task.isBlocked && (
+            <p tw="flex items-center text-coolGray-500 pt-1">
+              Task is blocked by other task.
+            </p>
+          )}
```

Comment:
Unfortunately this component is not displayed when it should because of a bug in develop that was discovered today